english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md

```diff
@@ -0,0 +1 @@
+Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
```

Comment:
My point is that I can still see the old link - `../breakfast/breakfast.md`. I can see both this link, and your fixed link, so I believe you need to remove this one.